.travis.yml

```diff
@@ -0,0 +1,29 @@
+# This config file for Travis CI
+sudo: required
+dist: trusty
+services:
+  - docker
+language: cpp
+cache: ccache
+compiler: gcc
+
+notifications:
+  email:
+    recipients:
+      - levi.armstrong@swri.org
+env:
+  global:
+    - ROS_DISTRO=kinetic
+    - UPSTREAM_WORKSPACE=file
+    - ROSINSTALL_FILENAME=dependencies.rosinstall
+    - TEST_BLACKLIST=bullet3_ros,fcl_ros,libccd_ros
+    - MAKEFLAGS="-j 4"
```

Comment:
Travis has sane defaults for this, and specifying something else might make this config brittle (ie: Travis may limit free accounts to a certain amount of resources and if we hardcode 4 jobs here, it could actually make things slower).

Did you set `4` here for a specific reason?

Reply:
`TEST_BLACKLIST` and `MAKEFLAGS` do not have any effect at all, because they are not used by `industrial_ci` and not passed to Docker.

Reply:
@ipa-mdl I have some external libraries that I would like not to run there tests. Is this possible with industrial_ci?

Reply:
At the moment there is only a whitelist option: `PKGS_DOWNSTREAM`